src/components/Displayer.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import RocketCard from './RocketCard';
+
+function Displayer(props) {
+  // eslint-disable-next-line react/prop-types
+  const { rockets } = props;
+  const [rocketsDisplay, setRocketsDisplay] = useState(null);
+  // Similar to componentDidMount and componentDidUpdate:
```

Comment:
Comments in code may be disliked by external code reviewers, just a reminder 😃

Reply:
Thanks for the feedback. Good that you pointed out those errors and have solved them already. I really appreciate it. :+1: